bsdd-graphql-soml-orig.yaml

```diff
@@ -0,0 +1,1149 @@
+        range: int
+        max: 1
+        nonNullable: false
+      dimensionMass:
+        name: dimensionMass
+        label: The Mass value of the dimension
+        range: int
+        max: 1
+        nonNullable: false
+      dimensionTime:
+        name: dimensionTime
+        label: The Time value of the dimension
+        range: int
+        max: 1
+        nonNullable: false
+      dimensionElectricCurrent:
+        name: dimensionElectricCurrent
+        label: The Electric current value of the dimension
+        range: int
+        max: 1
+        nonNullable: false
+      dimensionThermodynamicTemperature:
+        name: dimensionThermodynamicTemperature
+        label: The Thermodynamic temperature value of the dimension
+        range: int
+        max: 1
+        nonNullable: false
+      dimensionAmountOfSubstance:
+        name: dimensionAmountOfSubstance
+        label: The Amount of substance value of the dimension
+        range: int
+        max: 1
+        nonNullable: false
+      dimensionLuminousIntensity:
+        name: dimensionLuminousIntensity
+        label: The Luminous Intensity value of the dimension
+        range: int
+        max: 1
+        nonNullable: false
+      dynamicParameterPropertyCodes:
+        name: dynamicParameterPropertyCodes
+        label: List of codes of the properties which are parameters of the function
+          for a dynamic property. Only applicable for dynamic properties (IsDynamic)
+        range: string
+        max: inf
+        nonNullable: false
+      example:
+        name: example
+        label: Illustrate possible use or values of the Property
+        range: string
+        max: 1
+        nonNullable: false
+      isDynamic:
+        name: isDynamic
+        label: True if the value of this property is dependent on other properties
+          (as provided in DynamicParameterPropertyCodes)
+        range: boolean
+        max: 1
+        nonNullable: true
+      maxExclusive:
+        name: maxExclusive
+        label: Maximum value of the property, exclusive
+        range: decimal
+        max: 1
+        nonNullable: false
+      maxInclusive:
+        name: maxInclusive
+        label: Maximum value of the property, inclusive
+        range: decimal
+        max: 1
+        nonNullable: false
+      methodOfMeasurement:
+        name: methodOfMeasurement
+        label: Description of the method of measurement
+        range: string
+        max: 1
+        nonNullable: false
+      minExclusive:
+        name: minExclusive
+        label: Minimum value of the property, exclusive
+        range: decimal
+        max: 1
+        nonNullable: false
+      minInclusive:
+        name: minInclusive
+        label: Minimum value of the property, inclusive
+        range: decimal
+        max: 1
+        nonNullable: false
+      pattern:
+        name: pattern
+        label: 'An XML Schema Regular expression for the property value. See for explanation:
```

Comment:
If you put the string in single quotes, is the newline preserved?
Do we want to preserve the newline in this case?